VectorService.md

```diff
@@ -0,0 +1,142 @@
+VectorService
+=============
+
+#### Contents
+
+- [Description](#description)
+- [Request](#request)
+- [Request Structure](#request-structure)
+- [Sample Request](#sample-request)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+## Description
+
+This service provides vector representations for documents such as a Job or Resume, and for Carotene IDs with optional skills.
+
+
+The service is located at https://api.careerbuilder.com/core/document/vectors.  This service requires CareerBuilder OAuth 2.0 client credentials. For more information please see the authorization [documentation](/Readme.md#access).
+
+Requests can be sent in the form of GET or POST (form or json).
+
+## Request
+
+A request can be broken out into two distinct types identified through a parameter called `vector_type`. Available  `vector_type`'s are: **carotene_skills** and **semantic_fingerprint**.
+
+### Carotene Skills
+
+| Parameter  | Type | Required |  Description |
+|------------|------|----------|--------------|
+| carotene_id | String | True | A carotene ID as provided by our Job Classification services which uniquely identifies a job title.
+| carotene_version   | String | True | The version of carotene the ID is associated with.
```

Comment:
Are the valid values for carotene_version documented in some other place?

Reply:
ah they are not. I need to put that here. Good catch!